_config/current_narayana_docs.yml

```diff
@@ -1,4 +1,4 @@
 version: 5.0.4.Final
-date: 2014-11-10
+date: "2014-11-10"
```

Comment:
FYI This is unrelated to your change but the date and version need updating (I think the last time we generated the docs was for release 5.12.6.Final or 5.12.7.Final). We can investigate next time we update the  site).

Reply:
Noticed that. We should really create a script that updates that page automatically during the release process.

Reply:
I don't think that the `version` and `date` from this current_narayana_docs.yml file is used anywhere and if not then I think those two lines should be removed.

Our release script does modify current_narayana_release.yml: https://github.com/jbosstm/narayana/blob/master/build-release-pkgs.xml#L203-L209. I am not sure if `delete dir=` is correct in that block, I guess it should be `<delete file="${config-dir}current_narayana_release.yml"/>`

What do you think?

Reply:
+1

only
```
project: /docs/project/index.html
product: /docs/product/index.html
```
are used

Reply:
+1 also for the `delete file=`

I'll raise a PR to amend those two issues

Reply:
Thank you!